main/src/com/google/refine/model/medadata/validator/checks/Validator.java

```diff
@@ -0,0 +1,36 @@
+package com.google.refine.model.medadata.validator.checks;
```

Comment:
These validators look great to me, but I cannot figure out how to access them as a user. Is there any way to see them in the UI?

Reply:
okay you wrote in the PR that it's not implemented yet. fine.

Reply:
There is UT ValidateOperationTests.java to demonstrate the basic use of it. but we don't have the UI to trigger  this. It's non-trivial task. we can add it in future on top of current UI infra or defer until we separate the front end and have a modern UI framework.